### ISSUE TITLE
Improve support for Kotlin Native

### ISSUE THREAD

**Issue by rickclephas** [bugsnag/bugsnag-cocoa#1443]
### Description
I have been [playing](https://github.com/rickclephas/NSExceptionKt/blob/0686aa69b7f14dcaf6918441474784c171ad913b/nsexception-kt-bugsnag/README.md) with Bugsnag in a Kotlin Multiplatform iOS app.

So far it's working really well. I am just missing a single public API:

```objective-c
@interface BugsnagStackframe ()

+ (NSArray<BugsnagStackframe *> *)stackframesWithCallStackReturnAddresses:(NSArray<NSNumber *> *)callStackReturnAddresses;

@end
```

Once that API is public we can easily convert a Kotlin `Throwable` to a `BugsnagError`.

#### Describe the solution you'd like
For the `stackframesWithCallStackReturnAddresses` method to become part of the public API.

#### Describe alternatives you've considered
I am currently using the private API 🤪

**Comment by luke-belton** [comment]
Hi @rickclephas - thanks for raising this and glad to hear Bugsnag is working well in your Kotlin Multiplatform app! We're going to look at making your suggested change and will let you know when there are any updates 👍

**Comment by johnkiely1** [comment]
Hi @rickclephas , 
This has been released in v6.22.0.  
Thanks again for the suggestion.
